### DISCUSSION TITLE
blog/koh-lanta-2022-ambassadors-probability-problem/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#98] [General]
# blog/koh-lanta-2022-ambassadors-probability-problem/

What are the probability of winning for each ambassador in the television show Koh-Lanta 2022? Learn how to compute it in R through simulations and examples

https://statsandr.com/blog/koh-lanta-2022-ambassadors-probability-problem/

**Comment by reghais** [comment]
Thank you teacher

I hope that in the future, you will discuss compositional analysis , and discriminant analysis

Best

**Reply by AntoineSoetewey** [reply]
Hello @reghais,

Compositional analysis and discriminant analysis are not really within my field of expertise, so at the moment I do not plan to write about it.

I will let you know if it changes in the future. (If you want to write about it, feel free to let me know!)

Regards,
Antoine

**Reply by BenoitLondon** [reply]
Hi,
you could have computed the full winning probability directly
> P(Louana_win) = p_l_win + p_tie * p_l_win + p_tie^2 * p_l_win + p_tie^3 * p_l_win + ...
                        = p_l_win * (1 + p_tie +p_tie^2 + p_tie^3 + ...)
                        = p_l_win * 1 /(1 - p_tie) 
                        = q_c * p_l / (1 - q_c * q_l - p_c * p_l) 
                        = (2/3) * (1/2) / (1 -  (2/3) * (1/2) - (1/3) * (1/2)) 
                        = 2/3 = 0.6666...

thanks for the good read!

**Reply by AntoineSoetewey** [reply]
Correct, thanks for your input @BenoitLondon !
